website/docs/reference/exposure-properties.md

```diff
@@ -101,3 +104,30 @@ exposures:
 ```
 
 </File>
+
+#### Project-level configs
+
+You can define project-level configs for exposures in the `dbt_project.yml` file under the `exposures:` key using the `+` prefix:
+
+<File name="dbt_project.yml">
+
+```yml
+name: 'project_name'
+version: '1.0.0'
+config-version: 2
+
+.... rest of dbt_project.yml 
+
+exposures:
+  +enabled: true
+  +tags: ['docs']
+  +meta:
+    owner_team: analytics
+
+```
+
+</File>
+
+Note that only [`enabled`](/reference/resource-configs/enabled) is fully supported. Although `+tags` and `+meta` can appear under the config key of a compiled exposure, they don't populate the top-level `tags` or `meta` fields and as a result, tag-based [selection](/reference/resource-configs/tags#definition) or metadata extraction won't work or be available.
```

Comment:
Yeah, I'd rather just document `enabled`, since it's the only config that's fully supported